opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/FilterTasksPresenter.java

```diff
@@ -0,0 +1,116 @@
+package org.smartregister.reveal.presenter;
+
+import android.support.annotation.StringRes;
+
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.contract.FilterTasksContract;
+import org.smartregister.reveal.util.Constants.BusinessStatus;
+import org.smartregister.reveal.util.Constants.Intervention;
+import org.smartregister.reveal.util.Constants.InterventionType;
+import org.smartregister.reveal.util.PreferencesUtil;
+import org.smartregister.reveal.util.Utils;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Created by samuelgithengi on 12/18/19.
+ */
+public class FilterTasksPresenter implements FilterTasksContract.Presenter {
+
+
+    private FilterTasksContract.View view;
+
+    private Map<String, Integer> labelsMap;
+
+    private Map<String, Set<String>> checkedFilters = new HashMap<>();
+
+    private PreferencesUtil preferencesUtil;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'preferencesUtil' could be replaced by a local variable.](https://app.codacy.com/app/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)